mobile-workspace/latest/using/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Using Mobile Workspace
+---
+
+## Searching
+
+* Search is persistent and the Mobile Workspace allows global searching at any time. The results can be quickly filtered by files, folders or libraries just like in the Digital Workspace.
```

Comment:
Although I understand what you mean I wouldn't call search as being persistent in the docs I'd rather just say:
Global search is available from all of the main/top level screens allowing search at any time.

Reply:
Done